modules/test/playwright/package.json

```diff
@@ -17,8 +17,8 @@
 		"prettier": "2.1.2"
 	},
 	"scripts": {
-		"checkFormat": "liferay-npm-scripts prettier \"**/*.ts\" --list-different",
-		"format": "liferay-npm-scripts prettier \"**/*.ts\" --write",
+		"checkFormat": "tsc && liferay-npm-scripts prettier \"**/*.ts\" --list-different",
```

Comment:
Not sure if it makes sense to invoke `tsc` prior to `checkFormat` and `format` or only for `checkFormat` or if it would be better to have it in a separate script...

My guess is that we want to run it with format before sending PRs so that people don't break other people's tests.

Other than that, we may be happy with runtime execution failing when we run the tests.

Reply:
I think with our normal format scripts, we do the TS stuff in the "preflight" part of format. 

Not sure if you've looked, but is there any way we can just re-use `liferay-npm-scripts fix`?


If that isn't ideal, we should probably run it before so that prettier can format the d.ts files?

Reply:
OMG I forgot I had this PR in the queue :-( 

I've sent another one here -> https://github.com/liferay-frontend/liferay-portal/pull/3940

Reply:
I had a look at npm-scripts and I think they only run tsc on build so I didn't investigate the thing in more depth because this is urgent.

But it's true we should revisit npm-scripts after playwright is settled :+1: